### PR TITLE
Update Yarn 2 to 2.0.0-rc.36

### PR DESCRIPTION
**Pull request by JLHwung**
| Q                        | A 
| ------------------------ | ---
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/11779"></a>

Note that this PR targets to `next-8-dev`.

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -9,5 +9,6 @@ plugins:
 unsafeHttpWhitelist:
   - localhost
 
-# This path should always match the one in .yarnrc
-yarnPath: .yarn/releases/yarn-2.0.0-rc.33.js
+yarnPath: .yarn/releases/yarn-2.0.0-rc.36.js
+
+httpTimeout: 120000
```
Comment (JLHwung) [inline .yarnrc.yml:14]: `httpTimeout` is not available in Yarn 2.0.0-rc.33, so I update to the latest version.

**Comment by nicolo-ribaudo** [comment]
Oh did you notice a timed out build with yarn 2? I dropped it because I only saw the problem with yarn 1.

**Comment by JLHwung** [comment]
@nicolo-ribaudo Oh I thought it is dropped because it is not supported on yarn 2 rc.33. I am happy to remove `httpTimeout` if things are going well on gitpod.

**Comment by nicolo-ribaudo** [comment]
I didn't even notice that there wasn't a way to increment the timeout in the RC I was using 😅

If you saw a gitpod build failing then let's merge this PR, otherwise we can wait and see if the problem appears in the future.

**Comment by JLHwung** [comment]
@nicolo-ribaudo This PR has been revised to only update Yarn 2 from `rc.33` to `rc/36`, the lockfile format is changed a bit.

**Pull request merged**